CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 10.1.2 "pearl-parachute" (2020-08-03)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **testing:** emit pointer events on hover and mouseAway ([#20098](https://github.com/angular/components/issues/20098)) ([5b63309](https://github.com/angular/components/commit/5b63309)) |
+
+### material
+
+|            |                       |
+| ---------- | --------------------- |
+| bug fix |  **clipboard:** scroll position changing while copying on some browsers ([#20073](https://github.com/angular/components/issues/20073)) ([f58407d](https://github.com/angular/components/commit/f58407d)) |
+| bug fix |  **overlay:** only dispatch position change event if requested ([#19785](https://github.com/angular/components/issues/19785)) ([37b9664](https://github.com/angular/components/commit/37b9664)) |
+| bug fix |  **portal:** running change detection before nodes have been moved to outlet ([#16407](https://github.com/angular/components/issues/16407)) ([30da7e1](https://github.com/angular/components/commit/30da7e1)), closes [#16346](https://github.com/angular/components/issues/16346) |
+| bug fix |  **scrolling:** virtual scroll not accounting for margin when measuring range ([#19852](https://github.com/angular/components/issues/19852)) ([5892285](https://github.com/angular/components/commit/5892285)), closes [#19851](https://github.com/angular/components/issues/19851) |
```

Comment:
All of the ones above should be in CDK. They just didn't set an explicit package, so we fall back to `material`. 

We'll hopefully have linting for this soon, but it might help if we just not fall back to material, but add them to a section like `needs cleanup`.

Reply:
Fixed